pkg/sql/catalog/tabledesc/table_desc_builder.go

```diff
@@ -489,3 +493,15 @@ func maybeUpgradeIndexFormatVersion(idx *descpb.IndexDescriptor) (hasChanged boo
 	idx.Version = descpb.StrictIndexColumnIDGuaranteesVersion
 	return true
 }
+
+// maybeUpgradeNamespaceName deals with upgrading the name field of the
+// namespace table (30) to be "namespace" rather than "namespace2". This
+// occurs in clusters which were bootstrapped before 21.2 and have not
+// run the corresponding migration.
```

Comment:
Does this need a TODO comment to indicate the cluster version following which this can be removed?

Reply:
I figured that it would happen when the field it sets gets removed (that thing carries a TODO).